device_backends/LogicalNameMapping/src/LogicalNameMappingBackend.cc

```diff
@@ -11,7 +11,7 @@
 namespace ChimeraTK {
 
   LogicalNameMappingBackend::LogicalNameMappingBackend(std::string lmapFileName)
-  : hasParsed(false), _lmapFileName(lmapFileName) {
+  : hasParsed(false), _lmapFileName(std::move(lmapFileName)) {
```

Comment:
Hm,  have seen places where having an explicit move was preventing copy elision. I never know where to put it or not.

Reply:
This is a change suggested by the linter (see https://clang.llvm.org/extra/clang-tidy/checks/modernize/pass-by-value.html).
According to https://stackoverflow.com/questions/51705967/advantages-of-pass-by-value-and-stdmove-over-pass-by-reference this is more efficient than pass-by-reference, if an rvalue is passed as string.